src/main/java/backend/IssueMetadata.java

```diff
@@ -95,6 +102,10 @@ public boolean isUpdated() {
         return isUpdated;
     }
 
+    public String getIssueETag() {
```

Comment:
I wanted to make sure it won't be taken to mean one of the main 4 ETags in `UpdateSignature` (`issuesETag`, `milestonesETag`, `labelsETag`, `collaboratorsETag`). Is the method name a bit too redundant though?